MekHQ/src/mekhq/campaign/parts/Part.java

```diff
@@ -1275,6 +1275,10 @@ public int getQuantity() {
         return quantity;
     }
 
+    public int getSellableQuantity() {
+        return getQuantity();
```

Comment:
If we’re just returning the value of getQuantity() would it make more sense to just call getQuantity() directly instead of indirectly via a new method?

Reply:
Thanks for the feedback. The problem is getQuantity() is what you want for for selling most types of parts, but not others. Without this method, the code calling getQuantity() has to check what type of part they're dealing with, which breaks polymorphism and ends up being more brittle. With `getSellableQuantity()` in the base class, the calling code doesn't have to care what type it is--the responsibility of figuring out what the legal sellable quantity is is delegated to the part instead. 

If the caller doesn't care about selling parts at all it can continue using `getQuantity()` directly instead. Hope that helps answer your question!

Reply:
It flew straight over my head, but you’ve got a solid reason which is good enough for me - even if I don’t entirely understand it :D